build.gradle

```diff
@@ -30,8 +30,6 @@ allprojects {
         maven {
             url 'https://jitpack.io'
         }
-
```

Comment:
Remove these changes.
Only make changes in those files which are related to the issue this PR solves

Reply:
don't know whats happening in my system.. because these extra changes are not present in android studio.... should i close this PR and generate new after recloning the project...? may be these automated changes will not occur then